m2l31x/StdDriver/inc/spi.h

```diff
@@ -115,6 +115,24 @@ extern "C"
   */
 #define SPI_CLR_UNIT_TRANS_INT_FLAG(spi)   ((spi)->STATUS = SPI_STATUS_UNITIF_Msk)
 
+/**
+  * @brief      Disable Slave 3-wire mode.
+  * @param[in]  spi The pointer of the specified SPI module.
+  * @return     None.
+  * @details    Clear SLV3WIRE bit of SPI_SSCTL register to disable Slave 3-wire mode.
+  * \hideinitializer
+  */
+#define SPI_DISABLE_3WIRE_MODE(spi)   ( (spi)->SSCTL &= ~SPI_SSCTL_SLV3WIRE_Msk )
```

Comment:
```suggestion
#define SPI_DISABLE_3WIRE_MODE(spi)   ((spi)->SSCTL &= ~SPI_SSCTL_SLV3WIRE_Msk)
```

For coherency with the other definitions, same below

Reply:
yes, I modified the format for consistency.

Reply:
I still see them the same, sure you pushed the right thing?

Reply:
Sorry, miss to add this update before force push.
Now, fixed.